application/src/main/java/org/togetherjava/tjbot/commands/free/UserStrings.java

```diff
@@ -0,0 +1,55 @@
+package org.togetherjava.tjbot.commands.free;
+
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+
+/**
+ * Class containing all the strings sent to users during their interaction with the free command
+ * system. This does not include the logged strings or the exception strings.
+ */
+public enum UserStrings {
+    NEW_QUESTION("""
+            Thank you for asking a question in an available channel.
+            When a helper who can answer this question reads it they will help you.
+            Please be patient.
+
+            __Please do not post your question in other channels__
+            """),
+    MARK_AS_FREE("This channel is now free for a question to be asked."),
+    ALREADY_FREE_ERROR("This channel is already free, no changes made"),
+    NOT_READY_ERROR("Command not ready please try again in a minute"),
+    NOT_MONITORED_ERROR("This channel is not being monitored for free/busy status."
```

Comment:
Why not use text blocks here?

Reply:
Discord text blocks? Never heard about that.

Yes I mean Java text blocks.

Reply:
They dont need text blocks.
.....are u talking about java text blocks or discord text blocks?

I will change it for consistency if you like.

Reply:
so, change to text blocks for consistency?

Reply:
I'd do that yeah, but your decision in the end.

Reply:
Changed